src/Sarif.Multitool/ExportCommand.cs

```diff
@@ -0,0 +1,65 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+
+using Microsoft.CodeAnalysis.Sarif.Driver;
+using Microsoft.CodeAnalysis.Sarif.Multitool.Rules;
+
+namespace Microsoft.CodeAnalysis.Sarif.Multitool
+{
+    public class ExportCommand : CommandBase
+    {
+        private readonly IFileSystem _fileSystem;
+
+        public ExportCommand()
+        {
+            _fileSystem = new FileSystem();
+        }
+
+        public int Run(ExportOptions options)
+        {
+            try
+            {
+                var list = CompositionUtilities.GetExports<SarifValidationSkimmerBase>(
```

Comment:
>list [](start = 20, length = 4)

Let's call this `rules` since "list" doesn't tell the reader what's in the list.